### PR TITLE
[REM] account: remove tax_audit field

### PR DESCRIPTION
**Pull request by moerradi**
the tax_audit field was introduced in v13 and it served the purpose of providing users with improved visibility regarding the impact of move lines on report lines. The computation of the sign for each move line when resolving tax grid amounts has now been simplified using the formula (tax_tag_invert * sign of the tag * balance). Consequently, the tax_audit field has become redundant and is no longer required.

Task-id: 3196711

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/i18n/account.pot [hunk 1]
```diff
@@ -3812,13 +3812,6 @@ msgstr ""
 msgid "Computed Balance"
 msgstr ""
 
-#. module: account
-#: model:ir.model.fields,help:account.field_account_move_line__tax_audit
-msgid ""
-"Computed field, listing the tax grids impacted by this line, and the amount "
-"it applies to each of them."
-msgstr ""
-
```
Comment (rugo-odoo) [inline addons/account/i18n/account.pot:3821]: You don't need to this on any of the `.po` files on master. The way it works on **master** is that each time you have a new version, all the `.po` and `.pot` files are exported again and we create a transiflex project. So all unused fields are automatically removed.

File: addons/l10n_de/migrations/1.1/post-migrate_update_amls.py [hunk 2]
```diff
@@ -37,17 +37,3 @@ def migrate(cr, version):
...
-           SET tax_audit = REGEXP_REPLACE(tax_audit, '(?<=(^|\s))68:', '60:')
-          FROM (
-              SELECT aml.id as aml_id
-                FROM account_move_line aml
-                JOIN account_account_tag_account_move_line_rel aml_tag_rel ON aml_tag_rel.account_move_line_id = aml.id
-               WHERE aml_tag_rel.account_account_tag_id IN %s
-               ) aml
-         WHERE id = aml.aml_id
-        """, [tuple(tags_60.ids)]
-    )
```
Comment (rugo-odoo) [inline addons/l10n_de/migrations/1.1/post-migrate_update_amls.py:53]: You can not do that. We don't touch at previous upgrades scripts unless there was a mistake on them. What you need to do is create a new update script in which you remove the field `tax_audit`. (It is mentioned on the red runbot.)

File: addons/l10n_es/upgrades/15.0.5.0/post-61-tag-split-2021.py [hunk 3]
```diff
@@ -66,12 +66,3 @@ def get_taxes_from_templates(templates):
...
         """, [new_tag.id, tuple(tax_ids), old_tag.id])
-
-        # Fix tax audit string
-        cr.execute("""
-            UPDATE account_move_line aml
-            SET tax_audit = REPLACE(tax_audit, %s, %s)
-            FROM account_account_tag_account_move_line_rel aml_tag
-            WHERE aml_tag.account_move_line_id = aml.id
-            AND aml_tag.account_account_tag_id = %s
-        """, [old_tag.name, f"{new_tag.name}:", new_tag.id])
```
Comment (rugo-odoo) [inline addons/l10n_es/upgrades/15.0.5.0/post-61-tag-split-2021.py:77]: Same.

**Review by rugo-odoo — CHANGES_REQUESTED** [review]

**Review by rugo-odoo — CHANGES_REQUESTED** [review]
Also improve you pr message. Why are you removing it ? It doesn't need to be long or detailed but as it is, it's not enough.

**Comment by moerradi** [comment]
@rugo-odoo  finally XD

**Pull request closed without merging**